packages/server/src/test/java/software/uncharted/terarium/hmiserver/service/tasks/TaskServiceTest.java

```diff
@@ -337,6 +337,22 @@ public void testItCanSendAmrToMmtRequest() throws Exception {
 		log.info(new String(resp.getOutput()));
 	}
 
+	// @Test
+	@WithUserDetails(MockUser.URSULA)
+	public void testItCanSendSimplifyModelRequest() throws Exception {
+		final ClassPathResource resource = new ClassPathResource("mira/SIR_model_modified.json");
+		final String content = new String(Files.readAllBytes(resource.getFile().toPath()));
+
+		final TaskRequest req = new TaskRequest();
+		req.setType(TaskType.MIRA);
+		req.setScript("mira_task:simplify_model");
+		req.setInput(content.getBytes());
+
+		final TaskResponse resp = taskService.runTaskSync(req);
```

Comment:
What are you testing? There is no assertions in this test.

Reply:
That we are able to send the request. I am not certain as to why all of the tests are to exclusively check if we can kick the job off and nothing more but this is the same formatting as many of the tests in this file such as:
```
testItCanSendTextExtractionRequest
testItCanSendLatexToAMRRequest
testItCanSendGenerateModelLatexRequest
testItCanSendAmrToMmtRequest
```

Reply:
Okay, let's leave it.  But I would like to stay away from adding tests that by large, do not do anything - they will just bloat the CI-build time.

Reply:
Very reasonable. I figured it may be important for reasons I do not understand as there are many like it but that makes sense.
I will leave it for now